pinot-query-runtime/src/main/java/org/apache/pinot/query/service/dispatch/timeseries/TimeSeriesDispatchClient.java

```diff
@@ -29,16 +29,19 @@
 
 /**
  * Dispatch client used to dispatch a runnable plan to the server.
- * TODO: This shouldn't exist and we should re-use DispatchClient. TBD as part of multi-stage
+ * TODO(timeseries): This shouldn't exist and we should re-use DispatchClient. TBD as part of multi-stage
  *   engine integration.
  */
 public class TimeSeriesDispatchClient {
+  // TODO(timeseries): Note that time-series engine at present uses QueryServer for data transfer from server to broker.
+  //   This will be fixed as we integrate with MSE.
+  private static final int INBOUND_SIZE_LIMIT = 256 * 1024 * 1024;
```

Comment:
note: going through gRPC code and a demo to see if this can adverse perf impact.

Reply:
Took a look. I think it's better to run our internal load tests again after this PR is merged. I'll get back if there's a hit to scalability (scalability is important to us due to high QPS requirements).

Reply:
Hopefully once we move to MSE integration this issue of managing grpc size goes away.